addons/epic-anchors/AnchorsPanel/AnchorsMain.gd

```diff
@@ -1,18 +1,24 @@
 tool
 extends Control
 
+export var use_editor_theme_color : bool = true
+export var custom_hover_color := Color("f63c3c")
```

Comment:
Maybe these two could come from a project setting using ProjectSettings.get? Just so it isn't hardcoded. :)